saleor/graphql/checkout/tests/mutations/test_checkout_lines_update.py

```diff
@@ -1622,3 +1622,181 @@ def add_variants_to_checkout_wrapper(*args, **kwargs):
     assert len(data["errors"]) == 1
     assert data["errors"][0]["field"] == "lineId"
     assert data["errors"][0]["code"] == CheckoutErrorCode.GRAPHQL_ERROR.name
+
+
+def test_checkout_lines_update_with_metadata(
```

Comment:
Also for better readability and separating part of the test, I would suggest aligning with our contributing guide: https://github.com/saleor/saleor/blob/main/CONTRIBUTING.md#given-when-then.
It's a small change, but it significantly improves the readability of tests.